fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsController.kt

```diff
@@ -67,12 +67,12 @@ class TranslationsController(
 
         if (defaultFromLanguage == null) {
             defaultFromLanguage = detectedLanguages?.documentLangTag
-            logger.info("Setting translating to use a default 'from' of $defaultFromLanguage.")
+            logger.info("Setting translating to use a default 'from'.")
```

Comment:
Had thoughts that these values maybe shouldn't be logged, is it too specific?

Reply:
(Forgot use cases were used in reference browser, I assume they should stay for that reason.) #5279